test/tic_tac_toe_web/controllers/board_controller_test.exs

```diff
@@ -2,10 +2,10 @@ defmodule TicTacToeWeb.BoardControllerTest do
     use TicTacToeWeb.ConnCase
 
     alias TicTacToeWeb.BoardController, as: Controller
-
-    describe "post_move" do
+    
```

Comment:
Should there be a test for the controller's `new_game` method?

Reply:
added!